joss_paper_0.18/paper.bib

```diff
@@ -0,0 +1,38 @@
+@SciPy{,
+  author =    {Eric Jones and Travis Oliphant and Pearu Peterson and others},
```

Comment:
That's because Google Scholar keeps track of any changes, there's no such citable paper or item with a DOI anywhere